main.tf

```diff
@@ -25,23 +24,25 @@ module "vault_cluster" {
   # source = "git::git@github.com:hashicorp/terraform-google-vault.git//modules/vault-cluster?ref=v0.0.1"
   source = "modules/vault-cluster"
 
+  project = "${var.gcp_project}"
```

Comment:
Should this be:

```hcl
gcp_project_id = "${var.gcp_project_id}" 
```

Note that we should use the same var name in both examples and modules for consistency